modules/check_in/config/routes.rb

```diff
@@ -19,5 +19,6 @@
   namespace :v2, defaults: { format: :json } do
     resources :patient_check_ins, only: %i[show create]
     resources :sessions, only: %i[show create]
+    resources :pre_check_in, only: %i[show create]
```

Comment:
same as above, resource name by convention would be `:pre_check_ins`

Reply:
👍